doc/source/new-project-guide/level-3-tutorial.rst

```diff
@@ -24,7 +24,7 @@ Create a new project by running the following command:
 
      package create workspace
 
-You will then be asked to enter the ``project-name``. The default value is ``workspace_folder``.
+You will then be asked to enter the ``project-name``. The default value is ``data-analysis-project``.
```

Comment:
this is not correct.  We want the default value to be ``workspace_folder``, or maybe ``workspace-folder``, but in the example that we give them the user entered the value ``data-analysis-project`` to try and give an example of a possible usage.  In other words, as a user, I have a project which is some data analysis so I want a workspace called ``data-analysis-project`` to differentiate from another project I have which could be ``home-finances-project`` or something.  I never select the default value because I want to create different projects.

Reply:
Yes. Fixed.